apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -4,28 +4,42 @@
  * External dependencies
  */
 import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+import './style.scss';
 
 domReady( () => {
 	const { closeButtonLabel, closeButtonUrl, editorPostType } = fullSiteEditing;
 
 	const editPostHeaderInception = setInterval( () => {
-		const closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+		// cycle through interval until header toolbar is found
+		const toolbar = document.querySelector( '.edit-post-header__toolbar' );
 
-		if ( ! closeButton ) {
+		if ( ! toolbar ) {
 			return;
 		}
 		clearInterval( editPostHeaderInception );
 
+		// add components toolbar with override class name (original will be hidden in ./style.scss)
+		const componentsToolbar = document.createElement( 'div' );
+		componentsToolbar.className =
+			'components-toolbar edit-post-fullscreen-mode-close__toolbar edit-post-fullscreen-mode-close__toolbar__override';
+		toolbar.prepend( componentsToolbar );
+
+		// create custom close button and append to components toolbar
+		const newCloseButton = document.createElement( 'a' );
 		// When closing Template CPT (e.g. header) to navigate back to parent page.
 		if ( 'wp_template_part' === editorPostType && closeButtonUrl ) {
-			const newCloseButton = document.createElement( 'a' );
 			newCloseButton.href = closeButtonUrl;
 			newCloseButton.innerHTML = closeButtonLabel;
-			newCloseButton.className = 'components-button components-icon-button is-button is-default';
 			newCloseButton.setAttribute( 'aria-label', closeButtonLabel );
-
-			const parentContainer = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar' );
-			parentContainer.replaceChild( newCloseButton, closeButton );
+		} else if ( 'page' === editorPostType ) {
+			newCloseButton.href = 'edit.php?post_type=page';
```

Comment:
Did you look into whether or not `closeButtonUrl` would work in this context? I'm not sure if it's set correctly for this editor, but it's worth a check.

Reply:
Unfortunately it returns as `null` in this context.